sky/templates/vsphere-ray.yml.j2

```diff
@@ -0,0 +1,108 @@
+cluster_name: {{cluster_name_on_cloud}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.provision.vsphere
+  region: {{region}}
+  availability_zone: {{zones}}
+
+auth:
+  ssh_user: ecl
```

Comment:
nit: Do we want to make it a more general name, such as `ubuntu` or `vsphereuser`?

Reply:
Changed to `ubuntu`